src/components/calcite-input/calcite-input.scss

```diff
@@ -51,7 +51,7 @@
   & .calcite-input-prefix,
   & .calcite-input-suffix {
     padding: 16px;
```

Comment:
so this is a good example where `@apply` can be used to eliminate a lot of this, the entire rule can be written as:

```
& .calcite-input-prefix,
& .calcite-input-suffix {
  @apply text-1 p-1;
  height: 56px;
}
```

Doing it this way also demonstrates how `56px` is not actually a valid value from our spacing scale as it's halfway between `h-12` and `h-16` Things that aren't "sanctioned" become really obvious and should probably get updated anyways